src/AspNetCoreModuleV2/InProcessRequestHandler/inprocess/inprocessapplication.cpp

```diff
@@ -371,263 +336,14 @@ IN_PROCESS_APPLICATION::SetCallbackHandles(
     m_ShutdownHandlerContext = pvShutdownHandlerContext;
     m_AsyncCompletionHandler = async_completion_handler;
 
-    CloseStdErrHandles();
+    m_pLoggerProvider->NotifyStartupComplete();
```

Comment:
Do we want to continue redirecting to the file after this point?

Reply:
For pipes, no. For file yes. Hence I dispose in notify startup complete for pipes but not files.

Reply:
Why do we want it for files?

Reply:
The pipe logger only captures logs for startup. File output is always redirect until the application shuts down.